### ISSUE TITLE
Dynamically rendered image src in OneAR component 

### ISSUE THREAD

**Issue by joshacross** [joshacross/final-project#19]
`          <a-assets-item img id="AR" src={`/images/${currentProduct.thumbnail}`} alt={currentProduct.name} />
`
Ultimately, we want to pass in customized 3D GLTF objects. However, in the interim to setup functionality, we first need to dynamically send images to OneAR component based on currentProduct._id. If we trigger by product id, then I believe we would be able to pull into product.thumbnail (for now), but product.modelImage <-- is what we want in the end and would have 3D GLTF files under modelImage. 

Currently, QR code will trigger http://localhost:3000/products/:id/ar <-- 

My thoughts were useParams from react-router-dom and grab id so that it can Queried to pull image of from product by id. 

I started to pull in data similar to how details page is setup... but there has to be an easier way to do this.

**Comment by joshacross** [comment]
1.) Add switch case
2.) Cloned images into OneAR Images folder 
3.) Dynamically pulling in the image and using the direct path to the OneAR images folder
4.) Not rendering the image, instead rendering a White Sphere.

Could be a challenge with the switch case that I built. 

@JohnBanas can you take a stab at this. It is in feature/image-test branch off of develop

**Comment by JohnBanas** [comment]
Issue closed in branch feature/image-test, merged to develop. The a-asset-image component was unnecessary. We could not pass our variables into the JSX in the component, which was passed to another component through an id reference. Once we realized the a-sphere component had its own src that could be set directly without passing in a-asset-image, we were able to use our currentProduct (set locally with useState) to access a specific products property of thumbnail, which contained the image of that product.
